OrbitCore/LinuxPerfRingBuffer.h

```diff
@@ -36,12 +37,13 @@ class LinuxPerfRingBuffer
         // If the sizes are not the same, the memory layout defined in LinuxPerfEvent.h 
         // does not match the one found in the ring buffer.
         assert(
-            sizeof(record.ring_buffer_data) == a_Header.size && 
-            "Incorrect layout of the perf ring buffer data.\nPlease file a bug at https://github.com/pierricgimmig/orbitprofiler.\n"
+            sizeof(record.ring_buffer_data) == a_Header.size &&
+            "Incorrect layout of the perf ring buffer data.\n"
+            "Please file a bug at https://github.com/pierricgimmig/orbitprofiler.\n"
```

Comment:
Do we have this statement for all failed asserts?

Reply:
No, I'm not sure why this was added. I'll remove the second line.